src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export function LoginView(props) {
+  const [ username, setUsername ] = useState('');
+  const [ password, setPassword ] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(username, password);
+    /* Send a request to the server for authentication */
+    /* then call props.onLoggedIn(username) */
+    props.onLoggedIn(username);
+  };
+
+  return (
+    <form>
+      <label>
+        Username:
+        <input type="text" value={username} onChange={e => setUsername(e.target.value)} />
+      </label>
+      <label>
+        Password:
+        <input type="password" value={password} onChange={e => setPassword(e.target.value)} />
+      </label>
+      <button type="submit" onClick={handleSubmit}>Login</button>
+      <button type="submit" onClick={handleSubmit}>Register</button>
+    </form>
+  );
+}
+
+LoginView.propTypes = {
+  user: PropTypes.shape({
+    Username: PropTypes.string.isRequired,
+    Password: PropTypes.string.isRequired
+  }),
```

Comment:
Do we need the user as a prop? Am not sure but if its from the course then that's fine.